libs/application-generic/src/usecases/merge-preferences/merge-preferences.usecase.ts

```diff
@@ -44,9 +43,7 @@ export class MergePreferences {
       ...(isWorkflowPreferenceReadonly ? [] : subscriberPreferences),
     ];
 
-    const mergedPreferences = deepMerge(
-      preferencesList as DeepRequired<PreferencesEntity>[],
-    );
+    const mergedPreferences = merge({}, ...preferencesList);
```

Comment:
We assume that the potential issue of slow response times could be that we used the custom `deepMerge` code that was copied from the lib and had not been maintained for the last 2 years. We want to verify that assumption.